### PR TITLE
INTR-193 Update the banner card component

### PR DESCRIPTION
**Pull request by CamLamb**
- [x] JIRA ticket referenced in title
- [x] Title is clear and concise
- [x] Description gives any relevant detail
- [x] Tests are up to date
- [x] Documentation is up to date

Screenshot 2024-07-23 at 15 48 10
Screenshot 2024-07-23 at 15 48 17

### CODE REVIEW

File: src/dw_design_system/dwds/components/banner_card.scss [hunk 1]
```diff
@@ -1,4 +1,22 @@
 .dwds-banner-card {
+    --banner-card-background-colour: var(--dbt-blue);
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/components/banner_card.scss:2]: why have we started doing these very verbose style class names? I thought we were sticking with internal class names being concise and short, and variations as well ...?

File: src/dw_design_system/dwds/components/banner_card.scss [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+
+    &--red {
+        --banner-card-background-colour: var(--dbt-red);
+    }
+
+    background-color: var(--banner-card-background-colour) !important;
+    border-bottom: none !important;
```
Comment (marcelkornblum) [inline src/dw_design_system/dwds/components/banner_card.scss:9]: are the important delarations a result of the inheritance?

File: src/dw_design_system/dwds/components/banner_card.scss (hunk 1 above)
Comment (marcelkornblum) [reply src/dw_design_system/dwds/components/banner_card.scss:2]: sorry put this against a var, how embarrassing

Comment (marcelkornblum) [reply src/dw_design_system/dwds/components/banner_card.scss:2]: there are a few in this file later though

File: src/dw_design_system/dwds/components/banner_card.scss (hunk 2 above)
Comment (CamLamb) [reply src/dw_design_system/dwds/components/banner_card.scss]: Yeah, I don't love them. I imagine there is a way to get around using them

Comment (CamLamb) [reply src/dw_design_system/dwds/components/banner_card.scss]: I managed to get rid of the `background-color` important as that is hidden by the `--main` background. Still not sure about the `border-bottom`

Comment (CamLamb) [reply src/dw_design_system/dwds/components/banner_card.scss]: !importants are gone!

**Review by alsyx — APPROVED** [review]

**Pull request merged**